src/Bicep.LangServer/Providers/ParamsSymbolResolutionResult.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Semantics;
+using Bicep.Core.Syntax;
+using Bicep.LanguageServer.CompilationManager;
+
+namespace Bicep.LanguageServer.Providers
+{
+    public class ParamsSymbolResolutionResult
```

Comment:
Can you make it a [Record](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records)?